x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/SequenceMatcher.java

```diff
@@ -305,34 +305,42 @@ public void clear() {
         clearCircuitBreaker();
     }
 
```

Comment:
Add a comment to these two methods stating that they are protected for testing purposes.